### PR TITLE
Create Testimonials section

### PR DESCRIPTION
**Pull request by ohunazz**
Created testimonials section to show all the testimonials

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -240,5 +240,74 @@ <h2>Let's reserve <span>a table</span></h2>
...
+                <button class="btn-large-orange">View All</button>
+            </div>
+        </section>
+        <section id="testimonials-section">
+            <div class="testimonials-section__container">
+                <h2>What customers say</h2>
+                <div class="testimonials-section__testimonials">
+                    <div class="testimonials-section__testimonial">
+                        <img src="./assets/customer-1.png" alt="" />
+                        <h4>James Steven</h4>
```
Comment (kulievtech) [inline index.html]: Typo: The name of the person is not James Steven. It's actually James Maden.

File: styles.css [hunk 2]
```diff
@@ -290,3 +290,138 @@
     line-height: 40px;
     margin-bottom: 45px;
 }
+
+/* Chefs Section*/
+
+#chefs-section {
+    text-align: center;
```
Comment (kulievtech) [inline styles.css:297]: You also added a chefs section to the testimonials section. Please fix it.

Comment (ohunazz) [reply styles.css:297]: Good catch. I'll fix it

File: index.html (hunk 1 above)
Comment (ohunazz) [reply index.html]: ok got it

Comment (kulievtech) [reply index.html]: lgtm

**Review by kulievtech — APPROVED** [review]
Good to go. You can merge it into main.

**Pull request merged**